src/plugins/navigation/public/top_nav_menu/create_top_nav_menu.tsx

```diff
@@ -24,9 +23,9 @@ export function createTopNav(
     const config = (props.config || []).concat(relevantConfig);
 
     return (
-      <i18n.Context>
```

Comment:
ℹ️ I am changing this because it creates a conflict with EUI and the comma here
<img width="387" alt="image" src="https://user-images.githubusercontent.com/17003240/163410561-a34ff391-4bdf-408e-96f9-0a0b3da3a1d4.png">

was translated as [Object object]